src/core/logical_layers/renderers/GenericRendererPopoverProvider.tsx

```
@@ -0,0 +1,38 @@
+    )?.properties;
+
+    if (!featureProperties?.[this.paramName]) return null;
+
+    const content = featureProperties[this.paramName];
+
+    if (this.tooltipType === 'markdown') {
+      // Simple markdown rendering - can be enhanced with proper markdown library later
+      return (
+        <div
+          style={{ maxWidth: '300px' }}
+          dangerouslySetInnerHTML={{ __html: content }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)